apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -238,9 +287,25 @@ static void tc_task_task_create(void)
 	int pid;
 	const char *task_param[2] = { TEST_STRING, NULL };
 	g_callback = ERROR;
+
+	/* Inavlid priority value check */
+
+	pid = task_create("tc_task_create", SCHED_PRIORITY_MIN - 1, 1024, create_task, (char * const *)task_param);
+	TC_ASSERT_EQ("task_create", pid, ERROR);
+	TC_ASSERT_EQ("task_create", errno, EINVAL);
+
+	/* Regular functionality check */
+
 	pid = task_create("tc_task_create", SCHED_PRIORITY_MAX - 1, 1024, create_task, (char * const *)task_param);
 	TC_ASSERT_GT("task_create", pid, 0);
 	TC_ASSERT_EQ("task_create", g_callback, OK);
+
+	/* New task creation failure check due to maximum task creation limit (>CONFIG_MAX_TASKS)*/
+	pid = task_create("tc_maxtask", SCHED_PRIORITY_MAX - 1, 1024, max_task, (char * const *)NULL);
+	TC_ASSERT_GT("task_create", pid, OK);
+
+	sleep(SEC_3);
```

Comment:
As I left before, you didn't check any result of making max tasks. Please add checking logic of that with TC_ASSERT macro.

Reply:
I checked with schedforeach(), but i am finding a discrepency 
So i will update task_create TC in a new patch.